src/django/oar/settings.py

```diff
@@ -50,6 +50,7 @@
     'django.contrib.sessions',
     'django.contrib.sites',
     'django.contrib.messages',
+    'whitenoise.runserver_nostatic',
```

Comment:
The docs make it seem like this adds a hook for `runserver`. Is it giving us anything via the Gunicorn setup?

Reply:
No, my train of thought was that if someone does run `runserver` then this functionality might be helpful. Thinking about it again, it's just cruft.